proteus/mprans/PresInc.h

```diff
@@ -176,8 +183,16 @@ namespace proteus
 	{
 	  flux = 0.0;
 	  for(int I=0;I<nSpace;I++)
-            flux+= a*grad_potential[I]*n[I];
```

Comment:
this actually looked like a bug--the sign on the flux at open boundaries was wrong